File: 02comm/01uart_write.js
var SerialPort = require('serialport');
var port = new SerialPort("/dev/serial0", {baudRate:9600}, function(err){
	if(err){
		console.log('err:' + err.message);
		port = null;
	}
});
if(port){
	process.stdout.write('>');
	process.stdin.on('data', function(data){
		port.write(data+'\r');
		process.stdout.write('>');
	});
}

File: 02comm/01uart_write.py
import serial

port = serial.Serial("/dev/serial0", baudrate=9600, timeout=3.0)
try:
    if port.isOpen():
        print('serial writer is ready.')
    
    while True:
        line = input(">")
        print( line)
        port.write( (line + '\r\n').encode())
finally:
    port.close()